src/makeRobot.js

```diff
@@ -39,6 +39,67 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      // eslint-disable-next-line max-len
+      return `name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`;
```

Comment:
```suggestion
      return (`name: ${this.name} ` 
        + `chip version: ${this.version},` 
        + ` wheels: ${this.wheels}`
      );
```
Вместо того чтобы отключать линтер